# IDL struct ::niut::USER_STR
int32 id
int32 trackedId
niut_TIME_STR date_discovered
niut_TIME_STR date
niut_TRACK_STATE state
niut_SKELETON_STR skeleton
